internal/statestore/backfill.go

```diff
@@ -59,6 +61,8 @@ func (rb *redisBackend) CreateBackfill(ctx context.Context, backfill *pb.Backfil
 	if res.(int64) == 0 {
 		return status.Errorf(codes.AlreadyExists, "backfill already exists, id: %s", backfill.GetId())
 	}
+
+	acknowledgeBackfill(redisConn, backfill.GetId())
```

Comment:
you are ignoring error here

Reply:
yes, thanks for noticing, will update

Reply:
TestCreateBackfill should be updated according to this new behavior.